cogs/sendRandomPost.py

```diff
@@ -1,26 +1,24 @@
 import discord
-from discord import app_commands
 from discord.ext import commands
 
 from api import getRandomMarketingPost
 import io
 import aiohttp
 
 
-class sendRandomPostCog(commands.Cog):
-
+class sendRandomPostCog(commands.Cog, name="Send random post commands"):
```

Comment:
These commands no longer appear on Discord after syncing the command tree, probably because of the changed decorators.

[Read the section in the IT Guide on Syncing Commands for more info on when and how to sync](https://docs.google.com/document/d/1A5LoTSJLK4-awOx0jS7FKdfBnZkxtWP88n0WYlQQPUU/edit#heading=h.rwrenn4ud8yy)

Reply:
![image](https://user-images.githubusercontent.com/101443389/223882484-41ff4746-d62e-4a25-841c-8330f9fb267b.png)